server/neptune/workflows/internal/deploy/revision/queue/deployer.go

```diff
@@ -115,9 +116,16 @@ func (p *Deployer) runPostDeployTasks(ctx workflow.Context, deployment terraform
 }
 
 func (p *Deployer) startPRRevisionWorkflow(ctx workflow.Context, deployment terraform.DeploymentInfo) error {
-	version := workflow.GetVersion(ctx, version.SetPRRevision, workflow.DefaultVersion, 1)
-	if version == workflow.DefaultVersion {
+	version := workflow.GetVersion(ctx, version.SetPRRevision, workflow.DefaultVersion, 2)
+	switch version {
+	case workflow.DefaultVersion:
 		return nil
+
+	// Skip rebasing open PRs on Force Apply
+	case 2:
+		if deployment.Root.Trigger == terraformActivities.ManualTrigger {
```

Comment:
manual triggers can also just be retries or api driven requests, is this still valid?  I think you should check the branch instead.

Reply:
Also we should remove the versioning once we see that all workflows are new. cuz this is going to get confusing.

Reply:
Makes sense! I've updated the condition to check for the branch instead. Yes, I'll look into removing the versioning stuff  once old workflows have completed execution